subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheEnablementIntegrationTest.groovy

```diff
@@ -98,4 +100,46 @@ class ConfigurationCacheEnablementIntegrationTest extends AbstractConfigurationC
         "long option"     | DISABLE_CLI_OPT
         "system property" | DISABLE_SYS_PROP
     }
+
+
+    def "can enable configuration cache using incubating and final property variants"() {
+        given:
+        buildFile """
+        task check {
```

Comment:
Good catch - I meant to invoke it instead of `help`, and forgot to do so. I didn't notice it because even though it was not invoked, the task was eagerly configured which made the checks work as expected.

I changed the test to invoke the `check` task now.